src/components/WelcomePage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { TaskProvider } from "../Context/todoContext.tsx";
+import Tasks from "./Tasks.tsx"
+import AddTask from "./AddTask.tsx";
+import styles from '../styles/styles.module.css'
+import { User } from "src/@types/user";
+
+
+type Props = {
+    userToken: string; 
+    user: User;
+    onLogout: () => void;
+};
+
+const WelcomePage : React.FC<Props> = ({userToken, onLogout, user}) => {
+    const [on, setOn] = React.useState(false);
+    console.log(user)
```

Comment:
Remove it please.

Cand faci `push` la code, verifica sa nu lasi `console.log` din astea folosite pentru debug.